Some text clean up (#126)
IETF-TEAS-WG/actn-poi: draft-ietf-teas-actn-poi-applicability.md

a result, the operation of these complex packet and optical networks is
often siloed, as each technology domain requires specialized skill sets.

As a consequence, in many existing network deployments, packet and optical networks are engineered and operated independently.

This separation is inefficient for several reasons.

> First, integrating packet and optical

Reviewer: @danielkinguk : I like the original Michael/Italo text but your wordsmithing is very good (as usual). About path diversity , my interpretation is something that for sure is improving the service quality and so the text is fine.

> First, integrating packet and optical

Author: ```suggestion
Firstly, integrating packet and optical
```

networks can significantly reduce both capital expenditures (CAPEX) and
operational expenditures (OPEX).

> Second, multi-technology topology insights can optimize troubleshooting (e.g., alarm correlation) and enhance network operation (e.g., coordination of maintenance events). Additionally, detailed inventory and planning information can also improve service assurance quality, such as detecting constraint violations or lack of resource diversity.

Author: ```suggestion
Secondly, multi-technology topology insights can optimize troubleshooting (e.g., alarm correlation) and enhance network operation (e.g., coordination of maintenance events). Additionally, detailed inventory and planning information can also improve service assurance quality, such as detecting constraint violations or lack of resource diversity.
```

> Second, multi-technology topology insight can speed up troubleshooting (e.g., alarm correlation) and network operation (e.g., coordination of maintenance events), and detailed inventory and planning information can also improve service assurance quality (e.g., detection of diversity constraint violations).
> Third, multi-technology traffic engineering enables more efficient use of

Author: ```suggestion
Second, multi-technology topology insights can optimize troubleshooting (e.g., alarm correlation) and enhance network operation (e.g., coordination of maintenance events). Additionally, detailed inventory and planning information can also improve service assurance quality, such as detecting constraint violations or lack of resource diversity.
Thirdly, multi-technology traffic engineering enables more efficient use of
```
